### ISSUE TITLE
what's the unit of "Time points of data"

### ISSUE THREAD

**Issue by shenscore** [yutongo/TIGON#4]
Hi,

In your EMT example, I notice that the 0.1 refers to 8 hours. 

Is this a fixed value?

**Comment by YuchiQiu** [comment]
Yes, we scaled the time 8 hours to 0.1. You could scale the time to arbitrary unit when you prepare for the input data.
